www/templates/default/html/less/events.less

```diff
@@ -422,3 +422,85 @@
     }
   }
 }
+
+
+.unl-event-time,
+.unl-event-location {
+  margin-top: .18em;
+}
+
+
+.unl-event-teaser-ol {
+  --item-size: 13.32em;
+  grid-template-columns: repeat(auto-fit, minmax(var(--item-size), 1fr));
+}
+
+
+.unl-event-teaser {
+  display: grid;
+  grid-template: 'a b' auto 'a c' 1fr / $length-em-8 1fr;
```

Comment:
I am getting $length-em-8 is undefined when trying to compile this .less

Reply:
Oops! Will fix!